src/universal/.devcontainer/devcontainer.json

```
@@ -111,7 +111,7 @@
                 "go.toolsManagement.checkForUpdates": "local",
                 "go.useLanguageServer": true,
                 "go.gopath": "/usr/local/go/bin/go",
```

```suggestion
                "go.gopath": "/go",
```

Could you help fix this when you are at it? 😄 
We have been going back and forth with `GOPath` variable, but seems like this should be `/go` - syncing with the GO Feature. 
Reference - https://github.com/devcontainers/features/blob/main/src/go/devcontainer-feature.json#L30